.yarn/versions/abd5bdb6.yml

```diff
@@ -0,0 +1,2 @@
+declined:
+  - "@yarnpkg/plugin-workspace-tools"
```

Comment:
`@yarnpkg/plugin-workspace-tools` and `@yarnpkg/cli` should both be marked as "patch" and others as "declined"